src/Cli/InstallCommand.php

```diff
@@ -10,6 +10,14 @@
 
 class InstallCommand extends Command
 {
+    private $dockerInstaller;
+
+    public function __construct(DockerInstaller $dockerInstaller)
+    {
+        $this->dockerInstaller = $dockerInstaller;
+        parent::__construct();
```

Comment:
we are calling `setName`. But `configure` method is called in the parent constructor.